### DISCUSSION TITLE
Production deployment

### DISCUSSION THREAD

**Discussion by mCzolko** [react-native-tvos/react-native-tvos#777] [General Discussions]
I've created an RN tvOS app and I need to release it to the customer. How can I do that? I've been struggling with this for few days and there is not much of documentation out there.

**Comment by douglowder** [comment]
It's the same process as for iOS apps... have you shipped an iOS app before?

**Reply by mCzolko** [reply]
I did ship https://decosoft.app and https://eanx.app but these are iOS apps, the tvOS requires different provisioning and eas-cli is not giving it to me. I described the issue here: https://stackoverflow.com/questions/78943856/expo-provision-file-for-tvos

Sorry Doug for bothering you with this! You are doing excellent job and I have huge respect for you and your contributions!

**Reply by douglowder** [reply]
You can absolutely ask me this, after all I do work at Expo!

I recommend that you create the tvOS provisioning profile yourself manually, and then upload it to EAS through the web interface. If that has issues, you also have the option of using local credentials for EAS builds.

**Reply by mCzolko** [reply]
I've build it thru Xcode now. So I'm good to go :)
